web-services/common-util/src/main/java/datawave/security/authorization/DatawavePrincipal.java

```diff
@@ -29,7 +29,7 @@
     private final String username;
     private final DatawaveUser primaryUser;
     @XmlElement
-    private final Set<DatawaveUser> proxiedUsers = new LinkedHashSet<>();
+    private final List<DatawaveUser> proxiedUsers = new ArrayList<>();
```

Comment:
I don't think that it would be problematic and would prefer to keep the entries that are sent to us.  On the other hand, chanding from a Set to a List *may* change the XML or JSON representation (though probably not).  Changing back to LinkedHashSet.